wain-syntax-binary/src/leb128.rs

```diff
@@ -39,15 +39,8 @@ impl Leb128 for i32 {
 
 fn read_64(bytes: &[u8], signed: bool) -> Result<(u64, usize)> {
     let mut ret = 0;
-    let mut idx = 0;
 
-    loop {
-        if bytes.len() <= idx {
-            return Err(Box::new(ErrorKind::UnexpectedEof {
-                expected: "part of LEB128-encoded integer",
-            }));
-        }
-        let b = bytes[idx];
+    for (idx, b) in bytes.iter().cloned().enumerate() {
```

Comment:
`Copy` 実装してる型の場合は `copied` でお願いします

```suggestion
    for (idx, b) in bytes.iter().copied().enumerate() {
```